hypothesis-python/src/hypothesis/extra/pandas/impl.py

```diff
@@ -92,13 +104,33 @@ def elements_and_dtype(elements, dtype, source=None):
             f"Passed dtype={dtype!r} is a strategy, but we require a concrete dtype "
             "here.  See https://stackoverflow.com/q/74355937 for workaround patterns."
         )
-    dtype = try_convert(np.dtype, dtype, "dtype")
+
+    pd_dtype_map = {
+        t.name: t for t in getattr(IntegerDtype, "__subclasses__", lambda: [])()
+    }
+
+    # is_na_dtype = isinstance(dtype, str) and dtype[0].isupper()
+    is_na_dtype = False
+
+    if issubclass(type(dtype), IntegerDtype):
+        is_na_dtype = True
+        dtype = np.dtype(dtype.name.lower())
+    elif isinstance(dtype, str) and dtype in pd_dtype_map.keys():
+        is_na_dtype = True
+        dtype = pd_dtype_map[dtype]()
+        dtype = np.dtype(dtype.name.lower())
```

Comment:
This seems redundant?  I think you either mean to assign one to `np_dtype` (but we'd unconditionally overwrite that later), or we can just use

```suggestion
        dtype = np.dtype(dtype.lower())
```